### PR TITLE
Fix 404 page

### PR DESCRIPTION
**Pull request by srenatus**
- Let apache know where it is (via .htaccess)
- Remove reference to blog archive (show_meta => meta_information.html)
- Set `url` in _config.yml

The last point may need attention for staging/dev environments, but it
seemed reasonable to set it: it makes the google search function work,
too.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ description: "Refugees Emancipation Supporters Campaign #resp15"
 
 # This URL is the main adresse. Don't include a slash at the end.
 #
-url: ""
+url: "http://support.refugeesemancipation.com"
```
Comment (plutonik-a) [inline _config.yml:24]: I think, setting this URL here could cause broken links when deploying to staging or dev, don't you think?

**Comment by srenatus** [comment]
@plutonik-a Yep, probably. I still think that it should be set to the correct value for each one of the deployments, so I'd rather fix the problem for dev/staging than keep production "less than ideal" ;)

I'm a little confused, though, that http://jekyllrb.com/docs/configuration/ doesn't know about any "url" setting (as opposed to "baseurl"). So I guess it's an addition by the theme, but I haven't investigated yet why this may be necessary at all.

baseurl can be overridden on the command line, so it'd allow for not creating several yaml files for dev/staging.

**Comment by srenatus** [comment]
This just helped me a bit, clear and simple: https://byparker.com/blog/2014/clearing-up-confusion-around-baseurl/

**Comment by srenatus** [comment]
Ok, so there once _was_ a `--url` setting, but it was then deprecated: https://github.com/jekyll/jekyll/issues/2936

I'm all for creating even more config files, then...

**Pull request merged**